SnM/SnM_Notes.cpp

```diff
@@ -1804,3 +1979,31 @@ void NF_DoUpdateSWSMarkerRegionSubWindow()
 			w->ForceUpdateMkrRgnNameOrSub(g_notesType);
 	}
 }
+
+void NotesWnd::OnDroppedFiles(HDROP _h) {
+    int iFiles = DragQueryFile(_h, 0xFFFFFFFF, NULL, 0);
+    char fn[SNM_MAX_PATH] = "";
+    for (int i=0; i < iFiles; i++)
+    {
+        int n = DragQueryFile(_h, i, fn, sizeof(fn));
+        char* ext = strrchr(fn, '.');
```

Comment:
Segmentation fault when dragging a file without extension: `strrchr` returns null (see other comment for an alternative).